src/Samples/CSharp/Analyzers/CSharpAnalyzers/CSharpAnalyzers.Test/project.json

```diff
@@ -1,10 +1,7 @@
 {
-    "dependencies": {
-    },
-    "frameworks": {
-      "net46": {}
-    },
-    "runtimes": {
-      "win7": { }
-    }
+  "dependencies": {
+  },
+  "frameworks": {
+    "net46": { }
+  }
```

Comment:
Is the removal of the `runtimes` section actually necessary? What was it doing? @jasonmalinowski ?